### PR TITLE
Add presentational and containers component unit tests

### PR DESCRIPTION
**Pull request by Nasdan**

### CODE REVIEW

File: 14 TestComponent/src/components/about/spec/aboutPage.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+describe('AboutPage presentational component', () =>{
+    it('should renders a div element with class equals row and this div has 2 children h2 and Link', () => {
+        let aboutPageWrapper = shallow(<AboutPage />);
+
+        expect(aboutPageWrapper.contains(
+            <div className="row">
+              <h2>About Page</h2>
+              <Link to="/members">Members</Link>
+            </div>
```
Comment (brauliodiez) [inline 14 TestComponent/src/components/about/spec/aboutPage.spec.tsx]: Just check an h2 about page exists

File: 14 TestComponent/src/components/common/spec/header.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import Header from '../header';
+import {Link} from 'react-router';
+
+describe('Header presentational component', () =>{
+    it('should renders a div element with class equals row and this div has a nav like children', () => {
+        let headerWrapper = shallow(<Header />);
+
+        expect(headerWrapper.contains(
```
Comment (brauliodiez) [inline 14 TestComponent/src/components/common/spec/header.spec.tsx]: Just check the it contains the two expected links

File: 14 TestComponent/src/components/common/spec/spinner.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        expect(spinnerWrapper.contains(
+            <div className="spinnerWrap">
+              <div className="spinnerOverlay"></div>
+              <div className="vertical-offset">
+                <div id="spinner">
+                  Loading...
+                </div>
+              </div>
+            </div>
+        )).to.be.true;
```
Comment (brauliodiez) [inline 14 TestComponent/src/components/common/spec/spinner.spec.tsx]: Let's check that spinnerWrap is present (maybe the content could change in the future but this div will remain)

**Comment by brauliodiez** [comment]
Feedback added, change as well all the tests that use mocks to sinon.test to get automatic cleanup.

Give me a buzz when the pull is ready for a second review.

**Comment by Nasdan** [comment]
Changes done!

File: 14 TestComponent/src/components/member/spec/memberPage.spec.tsx [hunk 4]
```diff
@@ -0,0 +1,444 @@
...
+        });
+
+        expect(componentWillReceivePropsMock.calledOnce).to.be.true;
+        expect(toastrMock.calledOnce).to.be.false;
+    }).bind(this));
+
+    it('should calls to componentWillReceiveProps but does not calls to toastr.success()' +
+        'passing saveCompleted equals true and setting root component props with ' +
+        'saveCompleted property equals true', sinon.test(() => {
+        let sinon: Sinon.SinonStatic = this;
```
Comment (brauliodiez) [inline 14 TestComponent/src/components/member/spec/memberPage.spec.tsx]: Add a comment (just to get tsd intellisense)

File: 14 TestComponent/src/components/member/spec/memberPage.spec.tsx [hunk 5]
```diff
@@ -0,0 +1,444 @@
...
+            saveCompleted: false
+        });
+
+        expect(componentWillReceivePropsMock.calledOnce).to.be.true;
+        expect(toastrMock.calledOnce).to.be.false;
+    }).bind(this));
+
+    it('should calls to componentWillReceiveProps but does not calls to toastr.success()' +
+        'passing saveCompleted equals true and setting root component props with ' +
+        'saveCompleted property equals true', sinon.test(() => {
```
Comment (brauliodiez) [inline 14 TestComponent/src/components/member/spec/memberPage.spec.tsx]: Add a comment "sinson.test" make automatic cleanup and add a link explaining how it works, we will do this just in one place to help people know about this

**Pull request merged**